examples/camera/camera_background_processing.py

```diff
@@ -0,0 +1,21 @@
+from pitop.camera import Camera, pil_to_opencv
+from PIL import ImageDraw
+import cv2
+from signal import pause
+
+cam = Camera()
+
+
+def show_image_with_cross(im):
+    draw = ImageDraw.Draw(im)
+    draw.line((0, 0) + im.size, fill=128)
+    draw.line((0, im.size[1], im.size[0], 0), fill=128)
+    cv2.imshow('crossed', pil_to_opencv(im))
+    cv2.waitKey(1)
+
+
+# send every other frame to our processing function
+cam.start_handling_frames(show_image_with_cross, frame_interval=2)
```

Comment:
What is this actually doing?

Reply:
`start_handling_frames` registers the callback argument to be called with each new frame from the camera.

The API developed this way to match existing camera methods such as `cam.start_detecting_motion(on_motion)` and `cam.start_capturing_video()`. You can use multiple such methods simultaneously, as opposed to our other event driven apis of the form `cam.on_frame = show_image_with_cross`.

`start_handling_frames` is the generic form of these methods, such that you could write: `cam.start_handling_frames(pitop.camera.VideoCapture().process)` to do our video capture. 

In this example the `frame_interval` option is used to skip alternate frames. That is useful to reduce the framerate for this callback alone if it takes too long to process the frames, without reducing the overall framerate of the camera as it could be used simultaneously for other things.

There is some autodoc explanation of this but perhaps I should flesh it out, or explain more in these examples?